src/Vulkan/Silk.NET.Vulkan/Chain.g.tt

```diff
@@ -340,14 +340,43 @@ public abstract unsafe partial class Chain
     for (var i = 1; i <= maximumItems; i++)
     {
         var tList = TypeList(i);
+        var iBaseList = i == 1 ? string.Empty : $" : IChain<{TypeList(i - 1)}>";
```

Comment:
should this include `IDisposable` as a base interface for `IChain<THead>`? all concrete chain types inherit IDisposable so it may be worth formalizing that to enable `using` blocks for polymorphic chains

Reply:
Great suggestion, thanks!